Automation_Custom_Script.sh

```diff
@@ -18,11 +18,16 @@ G_CONFIG_INJECT "+ sailtrack-status" "+ sailtrack-status" /boot/dietpi/.dietpi-s
 G_CONFIG_INJECT "+ sailtrack-timesync" "+ sailtrack-timesync" /boot/dietpi/.dietpi-services_include_exclude
 G_CONFIG_INJECT "+ sailtrack-tileserver" "+ sailtrack-tileserver" /boot/dietpi/.dietpi-services_include_exclude
 G_CONFIG_INJECT "+ sailtrack-processor" "+ sailtrack-processor" /boot/dietpi/.dietpi-services_include_exclude
-G_EXEC /boot/dietpi/dietpi-services dietpi_controlled telegraf
-G_EXEC /boot/dietpi/dietpi-services dietpi_controlled sailtrack-status
-G_EXEC /boot/dietpi/dietpi-services dietpi_controlled sailtrack-timesync
-G_EXEC /boot/dietpi/dietpi-services dietpi_controlled sailtrack-tileserver
-G_EXEC /boot/dietpi/dietpi-services dietpi_controlled sailtrack-processor
+G_EXEC /boot/dietpi/dietpi-services enable telegraf
+G_EXEC /boot/dietpi/dietpi-services start telegraf
+G_EXEC /boot/dietpi/dietpi-services enable sailtrack-status
+G_EXEC /boot/dietpi/dietpi-services start sailtrack-status
+G_EXEC /boot/dietpi/dietpi-services enable sailtrack-timesync
+G_EXEC /boot/dietpi/dietpi-services start sailtrack-timesync
+G_EXEC /boot/dietpi/dietpi-services enable sailtrack-tileserver
+G_EXEC /boot/dietpi/dietpi-services start sailtrack-tileserver
+G_EXEC /boot/dietpi/dietpi-services enable sailtrack-processor
+G_EXEC /boot/dietpi/dietpi-services start sailtrack-processor
```

Comment:
I wouldn't start the SailTrack services immediately. At this point the setup is still in an unfinished state. The setup finishes with a restart, which will start them if they are enabled. Therefore, it makes more sense to just enable the services here and let them start at the next boot, when the setup is completed. Consider removing the start of the services:
```suggestion
G_EXEC /boot/dietpi/dietpi-services enable sailtrack-status
G_EXEC /boot/dietpi/dietpi-services enable sailtrack-timesync
G_EXEC /boot/dietpi/dietpi-services enable sailtrack-tileserver
G_EXEC /boot/dietpi/dietpi-services enable sailtrack-processor
```